compiler/rustc_codegen_ssa/src/back/write.rs

```diff
@@ -943,7 +948,10 @@ fn execute_copy_from_cache_work_item<B: ExtraBackendMethods>(
             output_path.display()
         );
         match link_or_copy(&source_file, &output_path) {
-            Ok(_) => Some(output_path),
+            Ok(_) => {
+                links_from_incr_cache.push(source_file);
```

Comment:
Shouldn't `links_from_incr_cache` be filled in for cg_clif too?

Reply:
Like that?